--- barcode-0.98/plessey.c~	2000-11-07 17:43:18.000000000 +0000
+++ barcode-0.98/plessey.c	2015-04-05 20:20:57.170658797 +0000
@@ -148,7 +148,7 @@
                         checkptr[i+j] ^= check[j];
     }
     for (i = 0; i < 8; i++) {
-        sprintf(ptr, patterns[checkptr[strlen(text) * 4 + i]]);
+        sprintf(ptr, "%s", patterns[checkptr[strlen(text) * 4 + i]]);
 	ptr += 2;
     }
     fprintf(stderr, "CRC: ");
